os/binfmt/libelf/libelf_addrenv.c

```diff
@@ -151,37 +151,37 @@ int elf_addrenv_alloc(FAR struct elf_loadinfo_s *loadinfo, size_t textsize, size
 
 #ifdef CONFIG_APP_BINARY_SEPARATION
 #ifdef CONFIG_OPTIMIZE_APP_RELOAD_TIME
+	uint32_t datamemsize = loadinfo->datasize + loadinfo->binp->ramsize;
```

Comment:
I think we'd better remain comments why we calculate like below.

Reply:
Done